site/styles/_meta-detail.scss

```diff
@@ -221,15 +229,20 @@ s72-element-switcher,
   }
 
   .meta-detail-synopsis {
+    margin-bottom: -4px;
     max-width: 728px;
+
+    @include media-breakpoint-up(xs) {
+      padding-bottom: 4px;
+    }
     @include media-breakpoint-up(xl) {
       max-width: 1100px;
     }
   }
 
   .meta-detail-synopsis,
   .meta-detail-switcher-tagline {
-    padding-top: 22px;
+    padding-top: 12px;
```

Comment:
Same thing here... I reckon this should be a margin-bottom on the preceding element.

Reply:
As per other comment beyond the scope of this task. Scope is supposed to be just spacing tweaks/adjustments.